manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -471,7 +471,6 @@ spec:
           - configmaps
           - endpoints
           - pods/log
-          - pods/proxy
```

Comment:
The Kiali Operator supports several previous versions of Kiali Server. For OSSM 1.1 we support Kiali Server v1.12 and for OSSM 2.0 we support  Kiali v1.24. Can you confirm that those two versions do not need this permission?

I looked in the [v1.12](https://github.com/kiali/kiali-operator/blob/master/roles/v1.12/kiali-deploy/templates/openshift/role.yaml) and [v1.24](https://github.com/kiali/kiali-operator/blob/master/roles/v1.24/kiali-deploy/templates/openshift/role.yaml) openshift templates and I do not see it, so I think we do not need this (since no supported version of the productized Kiali Server supports this) but I want to make sure.

Reply:
(note that if we do not need this, we need to backport this to the v1.36 branch - assuming this entire feature is going in v1.36)

Reply:
I do not see a "backport/1.36" label on this PR - that tells me this feature is NOT going into v1.36. If this is true, then you need to remove this change because kiali-ossm is going to still need this permission.

Reply:
yep, you are right. This is not going to be backported to the 1.36. So the 1.36 needs this permission to function. Reverting this deletion. Thanks